Test/test15/ModelTest.bpl.expect

```diff
@@ -19,6 +21,12 @@ tickleBool -> {
   true -> true
   else -> true
 }
+*** STATE <initial>
```

Comment:
What are these new lines? Are they just debug lines?

Reply:
There's a line somewhere
`error.Model.Write(Options.ModelWriter ?? errorInfo.Out);`
that can print the model without the model states having been initialised. A different invocation that would automatically initialise the states would be:
`error.PrintModel(Options.ModelWriter ?? errorInfo.Out, error);`

Because now we strictly initialise the model states, these model prints are slightly larger.